README.md

```diff
@@ -2,14 +2,16 @@
 
 This is a [Kodi](http://kodi.tv) PVR addon for connecting to a [tvheadend](https://tvheadend.org) backend.
 
+[![License: GPL v2](https://img.shields.io/badge/License-GPL%20v2-blue.svg)](LICENSE.md)
```

Comment:
```suggestion
[![License: GPL v2+](https://img.shields.io/badge/License-GPL%20v2+-blue.svg)](LICENSE.md)
```

Reply:
There and on other places is updated, applied this also to PVR addons where requests still open.